espflash/src/error.rs

```diff
@@ -96,6 +96,10 @@ pub enum Error {
     #[error("The provided bootloader binary is invalid")]
     InvalidBootloader,
 
+    #[error("Invalid byte sequence read from the serial port while trying to detect Boot Mode")]
```

Comment:
Do you think we should mention the possible failure case? I.e "this be caused by a xtal frequency mismatch" or something?

Reply:
Good idea, just added it on the help message